megamek/src/megamek/common/EntityListFile.java

```diff
@@ -746,7 +746,7 @@ public static void writeEntityList(Writer output, ArrayList<Entity> list) throws
                 output.write("\" " + MULParser.ATTR_QUIRKS + "=\"");
                 output.write(String.valueOf(entity.getQuirkList("::")));
             }
-            if (entity.getC3Master() != null) {
+            if ((entity.getC3Master() != null) && (entity.getGame() != null)) {
```

Comment:
looking at getC3Master() it can already fail when game == null, so these checks should be reversed, I think.

Reply:
Good catch

Reply:
Fixed now